opm/input/eclipse/share/keywords/000_Eclipse100/N/NETBALAN

```diff
@@ -30,7 +30,8 @@
       "item": 4,
       "name": "THP_CONVERGENCE_LIMIT",
       "value_type": "DOUBLE",
-      "default": 0.01
+      "default": 0.01,
+      "dimension" : "Pressure"
```

Comment:
We **might** have used it wrong, but the definition looks like a **fraction** of the group target rate, so it should not be dimensioned?

Reply:
Maybe we should change the name to be something like 
`MANIFOLD_TOLERANCE` or `AUTOCHOKE_TOLERANCE` something similar to avoid the confusion. `THP_CONVERGENCE_LIMIT` is indeed a confusing/misleading name.

Reply:
@GitPaean is right; item 4 of NETBALAN is a fraction, not a pressure.  It is a pure scalar that should not have a `dimension` entry.

Reply:
Agree, that is what is written in the OPM manual. But when I read computeWellGroupThp() and the usage of the thp_tolerance in the RegulaFalsiBisection algorithm it seems to me that it was a thp tolerance. But a more careful read of the code points to a correct usage. I will close this PR and use another tolerance for https://github.com/OPM/opm-simulators/pull/6175